lib/schemapack.js

```diff
@@ -245,7 +245,7 @@ class SchemaPack {
         schema
       );
 
-      if (!matchedPaths.length) {
+      if (!(matchedPaths && matchedPaths.length)) {
```

Comment:
@jazzyarchitects when will matched not be truthy? retVal is being initialized as `[]` in findMatchingRequestFromSchema.

Reply:
@abhijitkane This was happening when I was returning `null` when url was empty. Changed it to return `[]` instead of `null`